### DISCUSSION TITLE
What & where is the "JavaMoney OSS library"?

### DISCUSSION THREAD

**Discussion by ghferrari** [JavaMoney/jsr354-ri#422] [Q&A]
The [userguide](https://github.com/JavaMoney/jsr354-ri/blob/master/moneta-core/src/main/asciidoc/userguide.adoc) mentions an important companion package which it refers to as the "JavaMoney OSS library". What and where is this?

A related question: if I want to use bitcoin as a currency, do I need to set it up myself or is this already defined in the OSS library?

Many thanks for any help and kudos to the JavaMoney maintainers for this package.

**Comment by keilw** [comment]
It is https://github.com/JavaMoney/javamoney-lib.
